src/gradle-it/test-gradle-common-flow.groovy

```diff
@@ -5,34 +5,36 @@ this.metaClass.mixin baseScript
 
 println "Test test-gradle-common-flow.groovy" + "\n\n"
 
-def gradleExec = System.getenv("GRADLE_HOME")
-assert gradleExec?.trim() : "\"GRADLE_HOME\" environment variable need to be set"
-
 def targetPath = getTargetPath(project)
-def baseDir = targetPath.toString()
 def gradlePath = getGradlePath(project)
 
-def $gradle = System.getenv("gradle")
+def gradlewName;
+
+// Determine OS and appropriate gradlew executable
+if (System.getProperty("os.name").toLowerCase().contains("windows")) {
+    gradlewName = "gradlew.bat"
+} else {
+    gradlewName = "gradlew"
+}
 
-System.out.println($gradle)
+def gradleExec = gradlePath.resolve(gradlewName).toString();
 
 def username = "admin"
 def password = "password"
 
-def storageUrl = getStorageUrl()
-
+System.out.println(gradleExec)
 
-runCommand(gradlePath, String.format("$gradleExec clean upload -Dcredentials.username=%s -Dcredentials.password=%s -info",
+runCommand(gradlePath, String.format("$gradleExec upload -Dcredentials.username=%s -Dcredentials.password=%s",
                                     username,
                                     password))
 
 // check if dependency was resolved from Strongbox
-assert targetPath.resolve('strongbox-vault\\storages\\storage-common-proxies\\carlspring\\com\\fasterxml\\jackson\\core\\' +
-                          '\\jackson-databind\\2.9.4\\').resolve('jackson-databind-2.9.4.jar').toFile().exists();
+assert targetPath.resolve('strongbox-vault/storages/storage-common-proxies/carlspring/com/fasterxml/jackson/core/' +
```

Comment:
I have one question. Is this change necessary? I mean change one slash by another. If it has sense maybe it's better to write it in os independent way like targetPath.resolve("strongbox-vault").resolve("storages")...

Reply:
Yes, I specifically requested it, as it doesn't work under Linux. :)